docs/source/rabbitmq-tutorial/examples/7-publisher-confirms/publish_asynchronously.py

```diff
@@ -0,0 +1,45 @@
+import asyncio
+
+from aio_pika import Message, connect
+
+
+def getMessagesToPublish():
+    for i in range(10000):
+        yield f"Hello World {i}!".encode()
+
+def handle_confirm(confirmation):
+    try:
+        result = confirmation.result()
```

Comment:
Admittedly I didn't put a lot of time into it, but I couldn't figure out a good way to actually import `Basic.Ack` without going through `aiormq`, which was unexpected to me (maybe my expectations are wrong, though).
I expected something like `aio_pika.messages.Basic.Ack` or just `aio_pika.Basic.Ack` (what works is `aio_pika.message.aiormq.spec.Basic.Ack`, but that is unwieldy)
There is no example or documentation that uses `aiormq`, yet.

Reply:
I just remember why it's done that way, the point is that it just creates `asyncio.Future` instances, and you need to explicitly return them without creating a coroutine, this will give a good balance between heavy task creation if the user wants to pass it to `asyncio.wait`, or `asyncio.gather`.

So `asyncio.wait` is a really good way in this case to separate responses with ack from others. And we shouldn't forget about returning messages which will also be considered an error, and if the brocker passed the message body back, it will be in the exception instance.